deployer/orb.yaml

```diff
@@ -1,6 +1,85 @@
 version: 2.1
 description: "Tools for running deployment commands"
 
+commands:
+  validate-tag-for-pip:
```

Comment:
Now that you've split this out... this feels to me like it belongs in the `linter` orb instead. Thoughts?

EDIT: on second thought, maybe not. It depends, IMO, whether we think this is a general validation or very specific to Python. I'd argue that this fits mostly as `linter/validate-tag-is-semver` and just-so-happens to be using Python to do so, but perhaps `linter/validate-tag-is-python-semver` makes sense if it is this specific?

Reply:
Agreed, this should be linter level. The feedback loop is way too slow when coupled tightly to upload. The only issue is that the onus is on the developer to remember to use both the linter and the upload job in conjunction. That leaves the door open for issues. Thoughts on using both? i.e. move to linter, but also use it within the job here?

Reply:
I'm down for the "both" option :+1: